test/KillSwitchOracle.t.sol

```diff
@@ -0,0 +1,393 @@
+
+    address asset1 = makeAddr("asset1");
+    address asset2 = makeAddr("asset2");
+    address asset3 = makeAddr("asset3");
+    address asset4 = makeAddr("asset4");
+    address asset5 = makeAddr("asset5");
+
+    function setUp() public {
+        pool             = new MockPool();
+        poolConfigurator = new MockPoolConfigurator(IPool(address(pool)));
+        oracle           = new MockOracle(1e8);
+        anotherOracle    = new MockOracle(1e8);
+
+        killSwitchOracle = new KillSwitchOracle(
+            IPool(address(pool)),
+            IPoolConfigurator(address(poolConfigurator))
+        );
+        killSwitchOracle.transferOwnership(owner);
+    }
+
+    function test_constructor() public {
+        assertEq(address(killSwitchOracle.pool()),             address(pool));
+        assertEq(address(killSwitchOracle.poolConfigurator()), address(poolConfigurator));
+        assertEq(killSwitchOracle.triggered(),                 false);
+        assertEq(killSwitchOracle.numOracles(),                0);
+        assertEq(killSwitchOracle.owner(),                     owner);
+    }
+
+    function test_setOracle_revertOnlyOwner() public {
+        vm.expectRevert(abi.encodeWithSignature("OwnableUnauthorizedAccount(address)", randomAddress));
+        vm.prank(randomAddress);
+        killSwitchOracle.setOracle(address(oracle), 0.99e8);
+    }
+
+    function test_setOracle() public {
+        assertEq(killSwitchOracle.numOracles(),                      0);
+        assertEq(killSwitchOracle.oracleThresholds(address(oracle)), 0);
+
+        vm.expectEmit(address(killSwitchOracle));
+        emit SetOracle(address(oracle), 0.99e8);
+        vm.prank(owner);
+        killSwitchOracle.setOracle(address(oracle), 0.99e8);
+
+        assertEq(killSwitchOracle.numOracles(),                      1);
+        assertEq(killSwitchOracle.oracleAt(0),                       address(oracle));
+        assertEq(killSwitchOracle.oracleThresholds(address(oracle)), 0.99e8);
+    }
+
+    function test_setOracle_twice() public {
+        assertEq(killSwitchOracle.numOracles(),                      0);
+        assertEq(killSwitchOracle.oracleThresholds(address(oracle)), 0);
+
+        vm.prank(owner);
+        killSwitchOracle.setOracle(address(oracle), 0.99e8);
+
+        assertEq(killSwitchOracle.numOracles(),                      1);
+        assertEq(killSwitchOracle.oracleAt(0),                       address(oracle));
+        assertEq(killSwitchOracle.oracleThresholds(address(oracle)), 0.99e8);
+
+        // Should only update the threshold
+        vm.prank(owner);
+        killSwitchOracle.setOracle(address(oracle), 0.98e8);
+
+        assertEq(killSwitchOracle.numOracles(),                      1);
+        assertEq(killSwitchOracle.oracleAt(0),                       address(oracle));
+        assertEq(killSwitchOracle.oracleThresholds(address(oracle)), 0.98e8);
+    }
+
+    function test_disableOracle_revertOnlyOwner() public {
```

Comment:
The word "revert" in all of the reverting test cases seems redundant and we weren't using them before

Reply:
removed